packages/components/src/dashicon/index.js

```diff
@@ -900,7 +900,7 @@ export default class Dashicon extends Component {
 				height={ size }
 				viewBox="0 0 20 20"
 			>
-				<Path d={ path } />
+				<path d={ path } />
```

Comment:
Shouldn't this continue to be Path? in order to get the React-Native version?

Reply:
I think temporary it should remain as is.

I was playing with `createElement` version which would do the replacement behind the scenes, but with no luck so far. I hope to do some more explorations soon.